formula-android-tests/src/test/java/com/instacart/formula/FormulaFragmentTest.kt

```diff
@@ -350,20 +352,18 @@ class FormulaFragmentTest {
         }
     }
 
-    private fun activity(): TestFragmentActivity {
+    private fun activity(): TestFormulaActivity {
         return scenario.activity()
     }
 
     private fun activeContracts(): List<FragmentKey> {
-        return scenario.get {
-            lastState!!.activeIds.map { it.key }
-        }
+        return lastState!!.activeIds.map { it.key }
     }
 
     private fun assertVisibleContract(contract: FragmentKey) {
         assertNoDuplicates(contract)
         // TODO: would be best to test visibleState() however `FragmentFlowState.states` is empty
```

Comment:
Is this TODO still valid?

Reply:
Yep, just checked. The tests needs a refactor so that it could rely on `visibleOutput`